src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -115,7 +115,7 @@ public CompletionRegistrationOptions GetRegistrationOptions(CompletionCapability
             {
                 DocumentSelector = RazorDefaults.Selector,
                 ResolveProvider = true,
-                TriggerCharacters = new Container<string>("@", "<", ":"),
+                TriggerCharacters = new Container<string>("@", "<", ":", "_"),
```

Comment:
Ah, ya we shouldn't treat this as a trigger character. Technically prefixed `_` should be part of the identifier for the language meaning it should be no different than typing `DateTime`. I believe the root problem lies in our C# language-configuration where the ["identifier" / "word" pattern](https://github.com/dotnet/razor-tooling/blob/04ce320f6b9b903927a465853dbcf40563c4997f/src/Razor/src/Microsoft.VisualStudio.RazorExtension/csharp-language-configuration.json#L52) exists.